src/imaging.py

```diff
@@ -41,18 +41,23 @@
                     help='Latitude of location. Default = +44d13m29s')
 parser.add_argument('--longitude', type=str, nargs='?', const='-76d29m52s', default='-76d29m52s',
                     help='Longitude of location. Default = -76d29m52s')
-parser.add_argument('--image_width', type=int, nargs='?', const=0, default=8,
+parser.add_argument('--image_width', type=int, nargs='?', const=8, default=8,
                     help='Height of image in degrees (int). Default = 8')
-parser.add_argument('--image_height', type=int, nargs='?', const=0, default=8,
+parser.add_argument('--image_height', type=int, nargs='?', const=8, default=8,
                     help='Width of image in degrees (int). Default = 8')
+parser.add_argument('--sun_alt', type=float, nargs='?', const=400, default=400,
```

Comment:
I'm using a default value of '400' here. This would be outside of the range of what would ever be the starting position of the sun, so it would be easy to confirm that its not a user input.